Definitions.md

```diff
@@ -31,7 +31,7 @@ An n-dimensional collection of discrete *samples* whose domain is a regular disc
 Related terms: [*sample*](#sample), [*image*](#image), [*hierarchy*](#hierarchy)
 
 ### *image*
-A set of numbers intended to be displayed on a screen. Ancillary data structures may be required to display or interpret an *image* (such as a lookup table), but these are not part of the *image* itself. An *image* is often, but not necessarily, acquired by a sensor situated within an optical system. *Images* can be represented in compact forms, for example as a compressed sequence of bytes or as a discrete function over a finite domain, but these are not canonical uses of the word “image”, and the word “image” by itself should refer only to *arrays* and array-like data structures.
+A set of numbers intended to be displayed on a screen. It is also a set of values, each of which is associated with a position in a *domain*. Ancillary data structures may be required to display or interpret an *image* (such as a lookup table), but these are not part of the *image* itself. An *image* is often, but not necessarily, acquired by a sensor situated within an optical system. *Images* can be represented in compact forms, for example as a compressed sequence of bytes or as a discrete function over a finite domain, but these are not canonical uses of the word “image”, and the word “image” by itself should refer only to *arrays* and array-like data structures.
```

Comment:
```suggestion
A visual representation of a physical object or physical area of space.

A *digital image* can be represented by a set of numbers intended to be displayed on a screen. Each number is associated with a position in a *domain*. Ancillary data structures may be required to display or interpret a *digital image* (such as a lookup table), but these are not part of the *image* itself. An *image* is often, but not necessarily, acquired by a sensor situated within a physical system such as an optical light microscope.

*Digital images* can be represented in compact forms, for example as a compressed sequence of bytes or as a discrete function over a finite domain. In the data standards here, the word “image” by itself refers only to *arrays* and array-like data structures that represent a digital image rastered over space. Unless otherwise specified, this rastering occurs at regular equispaced intervals, the pixel pitch. 
```